packages/react/src/themes/teams-high-contrast/components/Tree/treeTitleVariables.ts

```diff
@@ -0,0 +1,7 @@
+import { TreeTitleVariables } from '../../../teams/components/Tree/treeTitleVariables'
```

Comment:
Can be removed:
https://github.com/stardust-ui/react/pull/1779/files#r319038491